cli/src/main/java/io/aklivity/zillabase/cli/config/ZillabaseAdminConfig.java

```diff
@@ -63,6 +63,8 @@ public final class ZillabaseAdminConfig
                       - strategy: topic
               udf:
                 - server: ${{env.UDF_SERVER}}
+                - server: ${{env.UDF_PYTHON_SERVER}}
```

Comment:
let's change the env variable from `UDF_SERVER` to `UDF_JAVA_SERVER` for clarity now that we have python too.